generators/chipyard/src/main/scala/TestHarness.scala

```diff
@@ -25,6 +29,47 @@ trait HasHarnessSignalReferences {
   def success: Bool
 }
 
+class HarnessClockInstantiator {
+  private var _clockMap: HashMap[String, (Double, ClockBundle)] = HashMap.empty
+
```

Comment:
In the future we might have multiple clocks in the harness as well, so this should generalize? Pretty cool

Reply:
Yup. This in theory should be able to provide clocks to anything in the harness that wants it (in theory because I haven't tested that use case past the current AXI domain stuff)